Cargo.lock

```diff
@@ -438,6 +438,16 @@ name = "itoa"
 version = "0.4.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
```

Comment:
It seems a few dependencies have newer versions and the `Cargo.lock` file is updated.